src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/steps/VotingStep/VotingStep.tsx

```diff
@@ -1,10 +1,104 @@
 import React, { FC } from 'react';
+import { formatText } from '~utils/intl';
+import Button from '~v5/shared/Button';
+
+import CardWithStatusText from '~v5/shared/CardWithStatusText';
+import ProgressBar from '~v5/shared/ProgressBar';
+import TextWithValue from './partials/TextWithValue';
+import FormButtonRadioButtons from '~v5/common/Fields/RadioButtons/ButtonRadioButtons/FormButtonRadioButtons';
+import { ActionForm } from '~shared/Fields';
+import { ActionTypes } from '~redux';
+import { TextWithValueItem } from './partials/TextWithValue/types';
+import { VOTING_THRESHOLD } from '~constants';
+import MemberReputation from '~common/Extensions/UserNavigation/partials/MemberReputation';
 
 const displayName =
   'v5.common.ActionSidebar.partials.motions.MotionSimplePayment.steps.VotingStep';
 
 const VotingStep: FC = () => {
-  return <div>VotingStep</div>;
+  const items: TextWithValueItem[] = [
+    {
+      key: '1',
+      leftColumn: formatText({ id: 'motion.votingStep.votingMethod' }),
+      rightColumn: <span>Reputation-weighted</span>,
```

Comment:
Is this span needed here? Also, the pair of props should be more like label and value and not left/right column